### PR TITLE
LINK-2046 | filter events by event status

### PR DESCRIPTION
**Pull request by jorilindell**
## Description
Add dropdown to event search panel to filter events by the event status.

## Closes
[LINK-2046](https://helsinkisolutionoffice.atlassian.net/browse/LINK-2046)

## Screenshots
Screenshot 2024-06-10 at 14 15 20

Screenshot 2024-06-10 at 14 15 47

[LINK-2046]: https://helsinkisolutionoffice.atlassian.net/browse/LINK-2046?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Comment by terovirtanen** [comment]
## e2e tests result is __success__ for https://linkedcomponents-pr408-ui.dev.hel.ninja :laughing::tada::tada::tada:

File: src/domain/app/i18n/sv.json [hunk 1]
```diff
@@ -1109,6 +1109,7 @@
     "searchPanel": {
       "buttonCreateNew": "Lägg till nytt evenemang",
       "buttonSearch": "Sök efter evenemang",
+      "labelEventStatus": "Evenemang status",
```
Comment (harriris-vincit) [inline src/domain/app/i18n/sv.json:1112]:
```suggestion
      "labelEventStatus": "Evenemangets status",
```

**Review by harriris-vincit — CHANGES_REQUESTED** [review]
It appears that the event status filter selection is not remembered like, for example, the publisher selection is: when opening an event for editing and returning to the published events list, the event status selection is back to the default state (= empty selection).

**Comment by terovirtanen** [comment]
**LINKEDCOMPONENTS-UI branch is deployed to platta: https://linkedcomponents-pr408-ui.dev.hel.ninja** :rocket::rocket::rocket:

**Comment by terovirtanen** [comment]
## e2e tests result is __failed__ for https://linkedcomponents-pr408-ui.dev.hel.ninja :crying_cat_face::anger::boom::boom:

File: src/domain/app/i18n/sv.json (hunk 1 above)
Comment (jorilindell) [reply src/domain/app/i18n/sv.json]: Changed as suggested

**Comment by jorilindell** [comment]
> It appears that the event status filter selection is not remembered like, for example, the publisher selection is: when opening an event for editing and returning to the published events list, the event status selection is back to the default state (= empty selection).

Good catch. The event status value was not initialised in the first load. Added an initial value for eventStatus and the bug should be fixed.

**Comment by terovirtanen** [comment]
**LINKEDCOMPONENTS-UI branch is deployed to platta: https://linkedcomponents-pr408-ui.dev.hel.ninja** :rocket::rocket::rocket:

**Comment by terovirtanen** [comment]
## e2e tests result is __failed__ for https://linkedcomponents-pr408-ui.dev.hel.ninja :crying_cat_face::anger::boom::boom:

**Review by harriris-vincit — APPROVED** [review]
Can confirm that now the event status filters are remembered. LGTM :+1:

**Pull request merged**